http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSManualTest.scala

```diff
@@ -0,0 +1,160 @@
+package org.broadinstitute.dsde.workbench.leonardo.util
+
+import cats.effect.std.Semaphore
+import cats.effect.{IO, Resource}
+import org.broadinstitute.dsde.workbench.azure._
+import org.broadinstitute.dsde.workbench.google2.KubernetesSerializableName.{NamespaceName, ServiceAccountName}
+import org.broadinstitute.dsde.workbench.leonardo.CloudContext.Azure
+import org.broadinstitute.dsde.workbench.leonardo.KubernetesTestData.{makeApp, makeKubeCluster, makeNodepool}
+import org.broadinstitute.dsde.workbench.leonardo.SamResourceId.AppSamResourceId
+import org.broadinstitute.dsde.workbench.leonardo.TestUtils.appContext
+import org.broadinstitute.dsde.workbench.leonardo.config.Config.{dbConcurrency, liquibaseConfig}
+import org.broadinstitute.dsde.workbench.leonardo.config.SamConfig
+import org.broadinstitute.dsde.workbench.leonardo.db.{DbReference, KubernetesServiceDbQueries, SaveKubernetesCluster, _}
+import org.broadinstitute.dsde.workbench.leonardo.http.ConfigReader
+import org.broadinstitute.dsde.workbench.leonardo.{
+  App,
+  AppName,
+  AppResources,
+  AppStatus,
+  AppType,
+  BatchAccountName,
+  CloudContext,
+  DefaultNodepool,
+  KubernetesClusterStatus,
+  LandingZoneResources,
+  ManagedIdentityName,
+  Namespace,
+  NamespaceId,
+  NodepoolStatus,
+  StorageAccountName,
+  SubnetName
+}
+import org.broadinstitute.dsp.{ChartName, HelmInterpreter, Release}
+import org.typelevel.log4cats.slf4j.Slf4jLogger
+
+import scala.concurrent.ExecutionContext
+
+/**
+ * Manual test for deploying CoA helm chart against an AKS cluster. Usage:
+
+ *   sbt "project http" test:console
+ *   import cats.effect.unsafe.implicits.global
+ *   import org.broadinstitute.dsde.workbench.leonardo.util.AKSManualTest
+ *   AKSManualTest.deployApp.unsafeRunSync
+ */
+object AKSManualTest {
```

Comment:
This is a "manual test" for testing against a real AKS cluster.

In time maybe we want to make a swatomation test, but this is useful for now.